### PR TITLE
Chore/format all task

### PR DESCRIPTION
**Pull request by BiniCodes**
- Add run prettier in all files task
- fix style / run prettier in useAutocomplete.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "test": "npm run test -ws --if-present",
+    "format-all": "prettier --write .",
```
Comment (plumdumpling) [inline package.json:11]: Do we really need to add this here? We shouldn't need this anymore once all files are formatted. The tests will fail if someone opens a pull request with unformatted code.

Comment (plumdumpling) [reply package.json:11]: I would expect everyone to have Prettier running in the IDE when developing. If someone is working without it, it's their job to run Prettier somehow. They can do this with `npx prettier --write .` or however they like, but I think it shouldn't be our task to care about this use case. Developers working with this project, set up how it's meant to be, will never need this task, that's why I think we don't need it here.

Comment (BiniCodes) [reply package.json:11]: okay, if you prefer it like this.

Comment (plumdumpling) [reply package.json]: Only if it makes sense for you too 😄

**Review by plumdumpling — APPROVED** [review]

**Pull request merged**